lib/src/geolocation.dart

```diff
@@ -118,6 +118,15 @@ class GeolocationResponse extends GoogleResponseStatus {
       json.containsKey('error') ? json['error']['message'] : null,
       Location.fromJson(json['location']),
       json['accuracy']);
+
+  @override
+  Map<String, dynamic> toJson() {
+//    var map = super.toJson();
```

Comment:
you can remove commented line :)

and explain why you don't call `super.toJson()`